tools/inference.py

```diff
@@ -24,11 +24,7 @@
 import cv2
 
 from anomalib.config import get_configurable_parameters
-from anomalib.core.model.inference import (
-    Inferencer,
-    OpenVINOInferencer,
-    TorchInferencer,
-)
+from anomalib.deploy.inference import Inferencer, OpenVINOInferencer, TorchInferencer
```

Comment:
As per the suggestion in #56, should we import openvino only if the weights are ".onnx", ".bin", ".xml"? This means we would need to import not at the top of the module, but we could perhaps ignore such errors.